sf-docker/special/sf-with-keycloak/run.sh

```diff
@@ -20,9 +20,6 @@ fi
 
 set -e
 
-# start dependencies of SF - PgSql, RMQ and minio
-chmod go+rx sf_dependencies/etc_rmq
-chmod go+r sf_dependencies/etc_rmq/*
```

Comment:
Why remove these lines?

Reply:
Check the original file. It was useless code.

Reply:
You're right, I was disoriented